### PR TITLE
Genres and css around movieTile

### PR DESCRIPTION
**Pull request by AdrianJar**

### CODE REVIEW

File: src/feature/MovieBrowser/Movies/List/movieListSlice.js [hunk 1]
```diff
@@ -5,7 +5,7 @@ const movieListSlice = createSlice({
     initialState: {
         status: "loading",
         movies: [],
-        genres: [],
+        genres: null,
```
Comment (KatarzynaZaloba) [inline src/feature/MovieBrowser/Movies/List/movieListSlice.js:8]: Great job, Adrian!
I am feeling embarrassed for my code with Peoplelist 😄 
image

File: src/feature/MovieBrowser/Movies/List/PopularMovies/Data/MovieTile/styled.js
Comment (KatarzynaZaloba) [inline src/feature/MovieBrowser/Movies/List/PopularMovies/Data/MovieTile/styled.js:1]: It looks great on the small devices too!
image

**Review by KatarzynaZaloba — APPROVED** [review]
I totally approve! 
Great job! 👍

**Comment by ghost** [comment]
Wow Adrien, I'm impressed!
Really good job!

**Review by Marianna-Weychan — APPROVED** [review]
This is a very difficult piece of logic, you really did a great job

**Pull request merged**